Frameworks/UIKit/UILabel.mm

```diff
@@ -293,6 +200,7 @@ - (void)_initUILabel {
     }
 
     _textBlock = textBlock;
+    [_textBlock setLineStackingStrategy:WXLineStackingStrategyBlockLineHeight];
```

Comment:
Yes, this is in _InitUILabel, thus default. 

---
In reply to: [105781932](https://github.com/Microsoft/WinObjC/pull/2178#discussion_r105781932) [](ancestors = 105781932)